src/containers/BookList.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Book from '../components/Book';
+import store from '../reducers';
+
+const BookList = () => {
+  const list = store.getState();
```

Comment:
its connect redux to store using `connect()` function  and it uses `mapStateToProps` to fetch books from the `store`